en/docs/fragments/dsetup.md

```diff
@@ -0,0 +1,6 @@
+In earlier versions, WSO2 Identity Server had the option to create databases automatically using the 
+-DSetup option.  **From [January 2018 onwards](https://wso2.com/products/carbon/release-matrix/), 
+WSO2 Identity Server has deprecated the `              -DSetup             `** option.
```

Comment:
do we need to mention this?

Reply:
The fragment is not pulled in any doc for now. Will retain the fragment just in case.